src/components/layout/__tests__/ButtonPlus.test.tsx

```diff
@@ -4,31 +4,25 @@ import "jest-styled-components";
 
 import ButtonPlus from "components/layout/ButtonPlus";
 
-describe("button default image", () => {
-  test("Button render", () => {
+describe("ButtonPlus component", () => {
+  test("Button render plus", () => {
```

Comment:
I recommend using "should" statements in single test case descriptions. So here: "Should render button with plus icon"